docs/v0.3/sdk/go/fssrz-index.md

```diff
@@ -6,35 +6,68 @@ slug: /fssrz/sdk-go
 
 <div class="status-badge">Technical Preview</div>
 
-Dagger is a cross-platform CI/CD engine with three defining features:
+## What is the Dagger Go SDK?
 
-1. **Portable**. Your pipelines run in containers so you get the same behavior on your local machine and in your CI environment.
-2. **Scriptable**. Develop pipelines in Go, Typescript, Python, or even a shell script. No niche language or proprietary YAML is required.
-3. **Extensible**. Each pipeline has an API. Pipelines can be shared and reused across projects, teams or the entire community.
+The Dagger Go SDK contains everything you need to develop CI/CD pipelines in Go, and run them on any OCI-compatible container runtime.
 
-The Dagger Go SDK enables you to communicate with the Dagger API by calling native Go functions. With this SDK, you can author your CI/CD pipeline entirely in Go, without needing to learn a new language or think about interoperability.
+Currently the Go SDK consists of:
 
-## Usage
+* A go library: `dagger.io/dagger`
+* A CLI (optional)
+* Sample apps
+* This documentation
 
-The following sequence diagram explains how the Dagger SDK can be used in the typical CI/CD process flow:
+In practice, everything is optional except the Go library.
+
+## Who is it for?
+
+The Dagger Go SDK may be a good fit if you are...
+
+* A Go developer wishing their CI pipelines were Go instead of YAML
+* A developer who needs CI/CD, and is looking for an excuse to learn Go
+* Your team's "designated devops person", hoping to replace a pile of artisanal scripts with something more powerful
+* A platform engineer writing custom Go tooling, with the goal unifying continuous delivery across organizational silos
+* A cloud-native developer advocate or solutions engineer, looking to demonstrate a complex integration on short notice
+
+The Dagger GO SDK may *not* be a good fit if you are...
+
+* A developer who doesn't know Go, and is not interested in learning it
+* A "designated devops person" who doesn't think of themselves as a developer - nothing makes you happier than a battle-hardened shell script.
+* Someone who loves writing YAML all day, thank you very much
+* A container skeptic: the less containers are involved, the happier you are.
+
+## How it works
 
 ```mermaid
 graph LR;
 
-tool["your Go tool"] -..-> goSDK["Dagger Go SDK"]
+subgraph program["Your Go program"]
+  lib["Go library"]
+end
+
+engine["Dagger Engine"]
 
+oci["OCI container runtime"]
 subgraph A["your build pipeline"]
   A1[" "] -.-> A2[" "] -.-> A3[" "]
 end
-subgraph B["your deploy pipeline"]
+subgraph B["your test pipeline"]
   B1[" "] -.-> B2[" "] -.-> B3[" "] -.-> B4[" "]
 end
-subgraph C["your test pipeline"]
+subgraph C["your deploy pipeline"]
   C1[" "] -.-> C2[" "] -.-> C3[" "] -.-> C4[" "]
 end
-goSDK -..-> A1 & B1 & C1
+
+lib -..-> engine -..-> oci -..-> A1 & B1 & C1
 ```
 
-## Next Steps
+1. Your Go program imports the Dagger go library
```

Comment:
Should we replace "go library" with "Go SDK" in this list?

Reply:
- Until yesterday I would have said “yes”
- But yesterday @aluzzardi shared concern about us maybe conflating “library” and “sdk” to the point of maybe not matching the conventions of Go developers. This is an attempt at reconciling Andrea’s concern while continuing to use the SDK terminology. 

TLDR: it’s still a fine point under discussion but shouldn’t affect the overall structure.